spec/models/questionnaire_version_spec.rb

```diff
@@ -0,0 +1,43 @@
+require "rails_helper"
+
+RSpec.describe QuestionnaireVersion, type: :model do
+  describe "#localized_layout" do
```

Comment:
This assumes that the layout is in the correct shape. We'll add validity checks before traversing through swagger (which are being added in [this PR](https://github.com/simpledotorg/simple-server/pull/4674))